app/assets/javascripts/reducers/timeline.js

```diff
@@ -39,6 +41,13 @@ const blocksInWeek = (blocks, weekId) => {
   return count;
 };
 
+const validateTitle = (title) => {
+  if ((typeof title === 'string' || title instanceof String) && title.length <= 20) {
+    return true;
+  }
+  return false;
+};
```

Comment:
You can just shorten this to:
```js
return (typeof title === 'string' || title instanceof String) && title.length <= 20)
```

What's the case where this would not be a string?